src/Analyzers/CSharp/CodeFixes/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -310,6 +309,15 @@ private static bool IsVarDesired(ITypeSymbol type, OptionSet options)
                 }
             }
 
+            newType = newType.WithoutTrivia().WithAdditionalAnnotations(Formatter.Annotation);
+            // We need trivia between the type declaration and designation or this will generate
+            // "out inti", but we might have trivia in the form of comments etc from the original
+            // designation and in those cases adding elastic trivia will break formatting.
+            if (!designation.HasLeadingTrivia)
+            {
+                newType = newType.WithAppendedTrailingTrivia(SyntaxFactory.ElasticSpace);
+            }
```

Comment:
question.  if you just add the ElasticAnnotation to the newType node itself, does that work?

Reply:
No, that doesn't fix the issue.